tests/MCryptCompatTest.php

```diff
@@ -546,6 +562,7 @@ public function testMiddleKey()
         $plaintext = 'a';
 
         $td = mcrypt_module_open('rijndael-128', '', 'cbc', '');
+        $this->expectTriggerError('phpseclib_set_key(): Key size of 20 has been changed to: 24', E_USER_WARNING);
```

Comment:
This is testing the *real* mcrypt (if it's available) and using it to kinda establish base behavior. The real mcrypt doesn't output an error and, even if it did, I'm pretty sure that error wouldn't mention phpseclib lol